tests/python-pytest/onnx/import/onnx_import_test.py

```diff
@@ -53,6 +53,30 @@
         'https://s3.amazonaws.com/onnx-mxnet/model-zoo/opset7/bvlc_reference_rcnn_ilsvrc13.tar.gz',
 }
 
+@with_seed()
+def test_broadcast():
+    """Test for broadcasting in onnx operators."""
+    input1 = np.random.rand(1, 3, 4, 5).astype("float32")
```

Comment:
Do we need to prepend the 1 ? Does the test pass with tensors of shape (3, 4, 5) and (5) ?

Reply:
ok